e2e/version_test.go

```diff
@@ -123,5 +123,8 @@ Provider:   %s
 }
 
 func getProviderVersion(env string) string {
-	return fmt.Sprintf("Snyk (%s (standalone))", os.Getenv(env))
+	if runtime.GOOS != "windows" {
```

Comment:
question: Why is this needed?

Reply:
I'm not entirely sure what changed (but I validated it on a windows machine) but on windows machine, the standalone version doesn't print anymore ` (standalone)` in the version, only on windows.
It's not a big issue, what we want to know here is we got the right version number.